### PR TITLE
issue #26: build and push to gcr, removed old deployment workflow

### LINKED ISSUE #26: Add github workflow to build and push the container to this organisation github registry and clean up of old workflows
- You can find the workflow at https://github.com/ai-cfia/github-workflows. It's called workflow-build-push-container-github-registry
- Clean up unused workflow inside this repository (if necessary)

### PR DESCRIPTION
**Pull request by ThomasCardin**
- Closes #26

### CODE REVIEW

File: .github/workflows/build-push-docker.yml
Comment (k-allagbe) [inline .github/workflows/build-push-docker.yml:16]: Once the branch 63 is merged, let's make sure this points to the main branch.

**Review by k-allagbe — CHANGES_REQUESTED** [review]

**Review by rngadam — APPROVED** [review]

File: .github/workflows/build-push-docker.yml
Comment (rngadam) [reply .github/workflows/build-push-docker.yml:16]: @k-allagbe since this is a future todo, should probably be a comment instead of a "request for changes" that blocks merging.

Comment (k-allagbe) [reply .github/workflows/build-push-docker.yml:16]: Okay got it. Are we okay merging with the unmerged workflow then?

Comment (rngadam) [reply .github/workflows/build-push-docker.yml:16]: IMO yes because having a pionter to the branch makes it clear that the called workflow is under development but OTOH we want to minimize blocking for deployment

**Pull request merged**
